data_steward/cdr_cleaner/manual_cleaning_rules/remove_operational_pii_fields.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+"""
+Some new operational fields exists, that were not blacklisted in the RDR export. These rows needs to be dropped in the 
```

Comment:
Possibly add a short summary and blank line before this paragraph.  https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings